aws/basic-eks-cluster/xebialabs/xlr-pipeline-ci-cd.yaml.tmpl

```diff
@@ -0,0 +1,68 @@
+{{$app := .AppName | kebabcase}}
```

Comment:
the `basic-aks-cluster` doesn't include an XLR pipeline. Should we remove from this one as well?

Reply:
it's needed due to the auth stuff